CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ The Singularity Project has been
 and re-branded as Apptainer.
 For older changes see the [archived Singularity change log](https://github.com/apptainer/singularity/blob/release-3.8/CHANGELOG.md).
 
+## Changes Since Last Release
+
+- Make `PS1` envrionment variable changeable via `%environment` section on
```

Comment:
Fix typo, `envrionment` => `environment`.

Reply:
Thank you @DrDaveD -san! I fixed typo.